workspaces/liferay-learn-workspace/client-extensions/liferay-learn-resources/LCP.json

```diff
@@ -0,0 +1,25 @@
+{
+	"cpu": 0.1,
+	"environments": {
+		"dev": {
+			"loadBalancer": {
+				"cdn": false,
+				"targetPort": 80
+			}
+		},
+		"infra": {
+			"deploy": false
+		}
+	},
+	"id": "liferaylearnresources",
```

Comment:
Could we just add in the image here (```"image": "liferay/caddy:latest",```) and delete the client-extension.yaml? Otherwise it messed up tilt.